src/CommandAndStateMessageParser.cpp

```diff
@@ -252,14 +252,14 @@ string CommandAndStateMessageParser::parseDriveRevolutionCommandMessage(
     string address,
     int model,
     commands::LinearAngular6DCommand command,
-    double minimum_vertical_command)
+    double vertical_command_offset)
 {
     auto message = payloadSetMessageTemplate(api_version, address, model);
     auto root = message["payload"]["devices"][address];
 
     auto vertical_cmd_sign = command.z() / abs(command.z());
     auto vertical_cmd =
-        max(abs(command.z()), minimum_vertical_command) * vertical_cmd_sign;
+        (abs(command.z()) + vertical_command_offset) * vertical_cmd_sign;
```

Comment:
Compensating for buoyancy should be done by adding a constant to the thrust, instead of having a minimum thrust command value, this is detrimental to the z control. (more details in here: https://app.shortcut.com/tidewise/story/19610/fix-buoyancy-compensation-in-deep-trekker-driver). 

I agree its not common to allow a negative constant offset. Do you think there is any possibility of it being useful? I don't see us putting the ROV anywhere near the water when its buoyancy is weak and it sinks. Im going to make these changes.

Reply:
I agree, just change the name to clarify that the offset is always absolute